server/sandbox_remove.go

```diff
@@ -98,7 +98,9 @@ func (s *Server) RemovePodSandbox(ctx context.Context, req *pb.RemovePodSandboxR
 	}
 
 	s.ReleasePodName(sb.Name())
-	s.removeSandbox(sb.ID())
+	if err := s.removeSandbox(sb.ID()); err != nil {
+		return nil, err
```

Comment:
same here.  Do we want to skip `s.PodIDIndex().Delete(sb.ID())`c on an error?

Reply:
I think so, switched to a log warning instead.